dask/dataframe/io/csv.py

```diff
@@ -786,7 +792,32 @@ def to_csv(
             [to_csv_chunk(d, f, **kwargs) for d, f in zip(dfs[1:], files[1:])]
         )
     if compute:
-        delayed(values).compute(scheduler=scheduler)
+        if compute_kwargs is None:
+            compute_kwargs = dict()
+
+        if scheduler is not None:
+            warn(
+                "passing 'scheduler' directly into `to_csv()` is deprecated and will be removed in a future version"
+                "pass the scheduler in as part of `compute_kwargs` instead."
+                f"e.g. df.to_csv(compute=True, compute_kwargs={{scheduler={scheduler}}})",
```

Comment:
Hope this isn't nitpicking too much

```suggestion
                "The 'scheduler' keyword argument for `to_csv()` is deprecated and will be removed in a future version. "
                "Please use the `compute_kwargs` argument instead. "
                f"For example, df.to_csv(..., compute_kwargs={{scheduler: {scheduler}}})",
```

Reply:
Not at all! I wasn't sure what kind of tone to aim for, thanks for the suggestion!